<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupWidget</class>
 <widget class="QWidget" name="GroupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>38</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(255, 255, 255);
font-size: 15pt;
}

QLabel#stateLabel{
background-color: rgb(255, 255, 255);
color:rgb(0, 63, 111);
}

QLabel#groupLabel{
background-color: rgb(255, 255, 255);
color:rgb(0, 63, 111);
}

QComboBox{
color:rgb(0, 63, 111);
border: 0px solid gray;
}

QPushButton#settingPushButton{
background-color: rgb(255, 255, 255);
color: rgb(0, 63, 111);
border-style: none;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="groupLabel">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Gr:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>NA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TOFD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PA</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="settingPushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Setting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="stateLabel">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
